stdlib/public/core/UnavailableStringAPIs.swift.gyb

```diff
@@ -109,3 +109,41 @@ ${stringSubscriptComment}
     Builtin.unreachable()
   }
 }
+
+% for View in ['Character', 'UnicodeScalar', 'UTF16', 'UTF8']:
+%   Index = 'String.%sView.Index' % View
+%   Distance = 'String.%sView.Index' % View
+extension ${Index} {
+  @available(
+    *, unavailable,
+    message: "To get the next index call 'index(after:)' on the instance of the ${View} string view that produced the index")
```

Comment:
These will end up saying something like:

    To get the next index call 'index(after:)' on the instance of the UTF8 string view that produced the index

But `Character`...`UTF8` aren't the right type and won't point them to the right place—I think I'd prefer to say:

    To get the next index call 'index(after:)' on the ${View}View instance that produced the index

or, more briefly

    To get the next index call 'index(after:)' on the related ${View}View instance